### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by GhausAliKhan**
In this branch I did: 

- Write unit tests with React Testing Library.
- Write unit tests with snapshots with Jest.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @GhausAliKhan 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pair programming project, that does not have commit from your partner 👍🏿

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by GhausAliKhan** [comment]
Hi @prolajumokeoni 
I wanted to let you know that I'm repeating this block and I don't have any partners assigned. So, this is the reason I have submitted a pair programming project, that does not have commit from my partner.

**Comment by GhausAliKhan** [comment]
I wanted to let you know that I'm repeating this block and I don't have any
partners assigned. So, this is the reason I have submitted a pair
programming project, that does not have commit from my partner.

**Comment by prolajumokeoni** [comment]
Hi @GhausAliKhan, thank you for pointing this out, Kindly add a screenshot of the approval  you got to work solo so that the next reviewer can confirm it 👍🏿

**Comment by GhausAliKhan** [comment]
image

File: src/testing/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  });
+  test('9 times 9', () => {
+    const result = operate(9, 9, 'x');
+    expect(result).toBe('81');
+  });
+  test('9 divided 9', () => {
+    const result = operate(9, 9, '/');
+    expect(result).toBe('1');
+  });
+});
```
Comment (Shedrack-Sunday) [inline src/testing/operate.test.js]: - [ ] Good job so far 👏 . Kindly add the modulus test ti make sure all edge cases are taken care of 👍 

### View:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @GhausAliKhan r          👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with the React Testing features ✔️
- Good work with the snapshots descriptions ✔️ 
    
### Confirmed view:
image

# Required Changes  🌵  
_Check the comments for a detailed review  on changes to be made

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @GhausAliKhan , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All test passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
